recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - jupyter_client
     - ipython
     - ipykernel
-    - prompt_toolkit >=2.0.0,<2.1.0
+    - prompt_toolkit >=2.0.0,<3.1.0
```

Comment:
I am not sure this works since we need to not include 3.0.0 or 3.0.1 from the setup.py